CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
-[[#]] Change Log
+# Changelog
 
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+### enhancements
+- leveraged reusable workflows for pipelines
```

Comment:
I wouldn't include the pipeline changes in the changelog (after all they shouldn't change the integration at all). I don't know if we could take the chance to update the golang version and include that instead.

Reply:
Should I be changing from 1.22.9 -> 1.23.2 ?

Reply:
Only if we are including some changes related to the code itself (dependency bumps, bug-fixes, features, ...). I'd say that changes such as pipelines or dev-dependencies (Eg: a testify upgrade) should not be included in the changelog and should not generate a new release

Reply:
removed pipeline changes from the changelog.md file